circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   node:
-    # version: 0.11.8
+    version: 7.2
```

Comment:
I'd be happy moving to an even more recent version, but I heard 7.2 is the one you settled with for now :)